packages/harmony/src/components/button/FilterButton/FilterButton.tsx

```diff
@@ -23,7 +17,7 @@ import {
 export const FilterButton = forwardRef<HTMLButtonElement, FilterButtonProps>(
   function FilterButton(props, ref) {
     const {
-      initialSelectionIndex,
+      selection: selectionProp,
```

Comment:
Does this break existing usage of FilterButton that was using `initialSelectionIndex`? Or did we update all of those as well?